### DISCUSSION TITLE
moonraker.conf

### DISCUSSION THREAD

**Discussion by timarland** [bassamanator/Sovol-SV06-firmware#25] [Q&A]
So if I have my setup of klipper/mainsail/moonraker all done up via kiauh, do I need to copy your moonraker.conf over that 'stock' one to make this work?  thanks!

**Comment by bassamanator** [comment]
You can just add 
```
[file_manager]
enable_object_processing: True
```
to your `moonraker.conf`. If you have a low-powered device such as a RPiZero, then make no changes at all to this file.

**Comment by timarland** [comment]
Cool - those two lines are added anywhere in the moonraker.conf?  I have an rpi A+ fwiw.  thanks!

**Reply by bassamanator** [reply]
Not sure what pi that is but anywhere at the bottom.

**Comment by timarland** [comment]
Awesome - thanks so much!  I'm going to try to flash later today.

Here's the the pi I have:  https://www.raspberrypi.com/products/raspberry-pi-3-model-a-plus/
